concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -988,40 +1032,76 @@ jobs:
-            - name: paas-haproxy-release
-            - name: aws-broker-boshrelease
-            - name: os-conf-boshrelease
-          run:
-            path: sh
-            args:
-              - -e
-              - -c
-              - |
-                ./paas-cf/concourse/scripts/bosh_login.sh {{bosh_fqdn}} bosh-secrets/bosh-secrets.yml
 
-                paas-cf/concourse/scripts/bosh_create_and_upload_release.rb graphite \
-                  {{cf_graphite_version}} graphite-statsd-boshrelease
+      - aggregate:
+        - task: upload-releases
+          config:
+            platform: linux
+            image: docker:///governmentpaas/bosh-cli
+            inputs:
+              - name: paas-cf
+              - name: bosh-secrets
+              - name: graphite-statsd-boshrelease
+              - name: grafana-boshrelease
+              - name: paas-haproxy-release
+              - name: aws-broker-boshrelease
+              - name: os-conf-boshrelease
+            run:
+              path: sh
+              args:
+                - -e
+                - -c
+                - |
+                  ./paas-cf/concourse/scripts/bosh_login.sh {{bosh_fqdn}} bosh-secrets/bosh-secrets.yml
+
+                  paas-cf/concourse/scripts/bosh_create_and_upload_release.rb graphite \
+                    {{cf_graphite_version}} graphite-statsd-boshrelease
 
-                paas-cf/concourse/scripts/bosh_create_and_upload_release.rb grafana \
-                  {{cf_grafana_version}} grafana-boshrelease
+                  paas-cf/concourse/scripts/bosh_create_and_upload_release.rb grafana \
+                    {{cf_grafana_version}} grafana-boshrelease
 
-                paas-cf/concourse/scripts/bosh_create_and_upload_release.rb paas-haproxy \
-                  {{cf-paas-haproxy-release-version}} paas-haproxy-release
+                  paas-cf/concourse/scripts/bosh_create_and_upload_release.rb paas-haproxy \
+                    {{cf-paas-haproxy-release-version}} paas-haproxy-release
 
-                paas-cf/concourse/scripts/bosh_create_and_upload_release.rb aws-broker \
-                {{cf_aws_broker_version}} aws-broker-boshrelease
+                  paas-cf/concourse/scripts/bosh_create_and_upload_release.rb os-conf \
+                    {{cf_os_conf_version}} os-conf-boshrelease
+
+                  paas-cf/concourse/scripts/bosh_create_and_upload_release.rb aws-broker \
+                    {{cf_aws_broker_version}} aws-broker-boshrelease
+
+        - task: get-and-upload-stemcell
+          config:
+            platform: linux
+            image: docker:///governmentpaas/bosh-cli
+            inputs:
+              - name: bosh-secrets
+              - name: paas-cf
+            run:
+              path: sh
+              args:
+                - -e
+                - -c
+                - |
+                  wget https://bosh.io/d/stemcells/bosh-aws-xen-hvm-ubuntu-trusty-go_agent?v={{stemcell_version}} -O stemcell.tgz
```

Comment:
That's minor but it would be better to get stemcell name from manifest instead hardcoding it here.
But that can be another story so we can figure out how to get all the required stemcells if we will decide to use multiple different stemcells/versions for jobs.